src/main/java/org/opentripplanner/street/model/vertex/Vertex.java

```diff
@@ -203,6 +203,23 @@ public List<StreetEdge> getOutgoingStreetEdges() {
     return result;
   }
 
+  /**
+   * Returns true if vertex is connected to another one by an edge
+   */
+  public boolean isConnected(Vertex v) {
+    for (Edge e : this.getOutgoing()) {
+      if (e.getToVertex() == v) {
+        return true;
+      }
+    }
+    for (Edge e : v.getOutgoing()) {
+      if (e.getToVertex() == this) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
I have one request that has nothing to do with your PR: I noticed that Vertex#azimuthTo is unused. Could I perhaps ask you to remove it?